src/views/MainHome.tsx

```diff
@@ -73,6 +73,13 @@ export default class MainHome extends Component<MainHomeProps, MainHomeState> {
             this.setState({ roomInvite: invite });
         });
 
+        socket.on(Constants.LEAVE_ROOM, (data: { roomid: string }) => {
```

Comment:
Why does `data` have to be an object with the `roomid` property? Can we make data just a string with `roomId`. Also, should we capitalize `i` in `roomid`?  Either way works for me, but we should just be consistent throughout the repo.

Reply:
Nice catch. Sorry I was kinda zoning out yesterday. And I'll leave it as `roomid` because everywhere else uses `roomid` but yes, in general, we should keep everything camelCase.

Reply:
Or should we change everything to camelcase? I think I've used camelcase in some cases.. sorry... We can just make that another task on clubhouse and do it as part of another pr tho

Reply:
Added to clubhouse!